spec/features/admin/loan_flow_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+feature 'loan flow' do
+  let(:division) { create(:division) }
+  let(:user) { create(:person, :with_member_access, :with_password, division: division).user }
+  let!(:loan) { create(:loan, division: division) }
+
+  before do
+    login_as(user, scope: :user)
+  end
+
+  # This should work, but for some reason it fails a lot more often
```

Comment:
Adding `wait` to the find calls did help, but I think in the end it was a race condition on `.loan-fields` there were actually 2 sections with that class, and the backbone view that was responsible for showing and hiding was attached to that class.